src/actions/l10n.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import type { Action, ThunkAction, Localization } from 'firefox-profiler/types';
+import {
+  fetchMessages,
+  lazilyParsedBundles,
+} from 'firefox-profiler/app-logic/l10n-bundling';
+import { negotiateLanguages } from '@fluent/langneg';
+import { ReactLocalization } from '@fluent/react';
+/**
```

Comment:
small nit: can you add a space between the imports and the function comment please? :)